src/mutual_inclusion_prototype/ReconstructedParticleStruct.jl

```diff
@@ -0,0 +1,18 @@
+using StaticArrays
+
+mutable struct ReconstructedParticle{Vector3fT, ClusterT, TrackT, ReconstructedParticleT, ParticleIDT, VertexT}
```

Comment:
```suggestion
mutable struct ReconstructedParticle{ClusterT, TrackT, ReconstructedParticleT, ParticleIDT, VertexT}
```
I think the `Vector3fT` can be dropped here, since that is not a relation but a component (in podio language). Components will always live in a separate file and (by podio) definition cannot be used in relations.